rust/worker/src/config.rs

```diff
@@ -164,12 +164,13 @@ mod tests {
                             connect_timeout_ms: 5000
                             request_timeout_ms: 1000
                     storage:
-                        S3:
-                            bucket: "chroma"
-                            credentials: Minio
-                            connect_timeout_ms: 5000
-                            request_timeout_ms: 1000
-                            upload_part_size_bytes: 8388608
+                        AdmissionControlledS3:
```

Comment:
note: we will have to upgrade the deployed config

Reply:
Yep, noted